src/utils/NetworkUtils.ts

```diff
@@ -123,6 +123,49 @@ export function chainIsL1(chainId: number): boolean {
   return [CHAIN_IDs.MAINNET, CHAIN_IDs.SEPOLIA].includes(chainId);
 }
 
+/**
+ * Determines whether a chain ID runs on an EVM-like execution layer.
+ * @param chainId Chain ID to evaluate.
+ * @returns True if chain corresponding to chainId has an EVM-like execution layer.
+ */
+export function chainIsEvm(chainId: number): boolean {
+  const chainFamily = PUBLIC_NETWORKS[chainId].family;
+  const checkEvmExceptionChains = (_chainId: number) => {
+    const {
+      MAINNET,
+      SEPOLIA,
+      ARBITRUM,
+      ARBITRUM_SEPOLIA,
+      LINEA,
+      POLYGON,
+      POLYGON_AMOY,
+      SCROLL,
+      SCROLL_SEPOLIA,
+      ZK_SYNC,
+      ZK_SYNC_SEPOLIA,
+    } = CHAIN_IDs;
+    return [
```

Comment:
Should this just return true unless the chain ID is solana for now? In other words, it should always return true currently?

Reply:
Yes. I didn't do that since we don't have Solana in constants yet.

Reply:
maybe just return true for now then and add a note that we can add solana chain in here once we have it avaialble